### PR TITLE
Bump rxjs to 7 along with akita/ini

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.org/wp/42235

Bumps:

- rxjs to 7
- akita and ini (following dependabot)

Updates reactivestates to rxjs by forking

### CODE REVIEW

File: frontend/src/app/features/boards/board/board.service.ts [hunk 1]
```diff
@@ -48,11 +55,13 @@ export class BoardService {
       return this.loadAllPromise;
     }
 
-    return this.loadAllPromise = this
-      .apiV3Service
-      .boards
-      .allInScope(projectIdentifier!)
-      .toPromise();
+    // eslint-disable-next-line no-return-assign
```
Comment (b12f) [inline frontend/src/app/features/boards/board/board.service.ts]: I see why you'd do this, we can discuss changing this rule, but tbh I'd also prefer assign first, return second here.

Comment (oliverguenther) [reply frontend/src/app/features/boards/board/board.service.ts]: Yeah that was just lazy, I'll fix it

File: frontend/src/app/shared/components/fields/macros/attribute-label-macro.component.ts [hunk 2]
```diff
@@ -78,36 +80,40 @@ export class AttributeLabelMacroComponent {
...
   }
 
-  private async loadResourceAttribute(model:SupportedAttributeModels, id:string, attributeName:string) {
+  private async loadResourceAttribute(model:SupportedAttributeModels, id:string, attributeName:string):Promise<void> {
     let resource:HalResource|null;
 
     try {
-      this.resource = resource = await this.resourceLoader.require(model, id);
+      // eslint-disable-next-line no-multi-assign
+      this.resource = resource = await firstValueFrom(this.resourceLoader.require(model, id));
```
Comment (b12f) [inline frontend/src/app/shared/components/fields/macros/attribute-label-macro.component.ts]: Same here, I'd just stick to the rule

**Review by b12f — APPROVED** [review]
LGTM. It's a bit sad that we loose the git history from the vendored reactivestates repo, but I guess it doesn't matter too much

**Comment by oliverguenther** [comment]
> LGTM. It's a bit sad that we loose the git history from the vendored reactivestates repo, but I guess it doesn't matter too much

I forked the project instead to our npm org. Even fixed the tests and circulare deps while at it!

**Pull request merged**